homeassistant/components/plugwise/config_flow.py

```diff
@@ -154,6 +161,30 @@ def is_matching(self, other_flow: Self) -> bool:
 
         return False
 
+    async def _verify_connection(
+        self, user_input: dict[str, Any]
+    ) -> tuple[Smile | None, dict[str, str]]:
```

Comment:
```suggestion
    ) -> Smile | dict[str, str]]:
```
this never returns both the api and an error, so we don't need a tupple.

Reply:
Thanks for spotting that, that was a left-over from tinkering (with a tuple).

Reply:
Its more about conveying intention directly on the method's signature. With a tuple it looks like it returns both Smile + a dict, while in fact it returns either one or the other. The Union represents that better than a Tuple.